src/misc/scan.c

```diff
@@ -579,12 +578,9 @@ int igraph_local_scan_1_ecount_them(const igraph_t *us, const igraph_t *them,
 
 /**
  * \function igraph_local_scan_k_ecount
- * Local scan-statistics, general function, edge count and sum of weights
- *
- * Count the number of edges or the sum the edge weights in the
- * k-neighborhood of vertices.
```

Comment:
Was this comment incorrect? I notice you removed any mention of weights.

Reply:
I found the brief description confusing, so I changed it, but then it's equal to the other description, so I removed that. I think this sentence captures what the function does. Maybe the new version should say "edge weights" instead of just "weights".

The weights are still mentioned :slightly_smiling_face:

Reply:
I think the new version is okay.